app/controllers/api/integrations_controller.rb

```diff
@@ -66,15 +65,15 @@ def update_production
 
     done_step(:apicast_gateway_deployed, final_step=true) if ApiClassificationService.test(@proxy.api_backend).real_api?
 
-    redirect_to action: :edit, anchor: 'proxy'
+    redirect_to action: :show, anchor: 'proxy'
```

Comment:
The problem of redirecting here to a legit action is that then it seems like this is also a reachable legit path, and it isn't 😄 so I propose removing it directly since it is not used anymore (for APIAP) 😉 Proposal pointing to this PR in https://github.com/3scale/porta/pull/2293

Alternatively, if you do not like my proposal, I would prefer if you add here a 'TODO' comment for it and instead of redirecting to a legit action, just do `render head: :success, nothing: true` or even `render json: {success: true}` is an acceptable middle-step.